Update agbinator.py

Igoreshazhu opened the pull request:
Added sound drivers by Full Fat and Software Creations.

Igoreshazhu committed:
Added sound drivers by Full Fat and Software Creations and new Krawall signature.

Igoreshazhu committed:
Update agbinator.py

Removed new Krawall signature. I'm not sure that Barbie games uses uncommercial version of Krawall. It should be checked.

Igoreshazhu committed:
Update agbinator.py

Returned new Krawall signature

loveemu commented:
I have not verified the validity of the signature but would like to approve this PR.

However, it appears that conflicts between branches must be resolved before a merge can be performed.

Igoreshazhu commented:
Well, this is not easy.

Igoreshazhu committed:
Update agbinator.py

Removed Krawall signature due the false positives; fixed some algorithms; added new 6 drivers

Igoreshazhu committed:
Merge branch 'master' into master
